db/helpers/common.js

```diff
@@ -32,6 +32,28 @@ const changeType = {
 };
 
 /**
+ * Returns duplicate strings in an given flat array
+ *
+ * ie. [a, b, c, c, a] => [a, c]
+ *
+ * @param {Array} arr Array of strings
+ * @returns {Array} Array of duplicate strings
+ */
+function findDuplicatesInStringArray(arr) {
```

Comment:
Wondering if we can do something like, adding the elements of the array to the set and check if the length of the set and the array are same and return true or false based on this.?